src/services/dashboard.js

```diff
@@ -53,8 +54,9 @@ export async function queryResults(params) {
       endpoints.run_index,
       selectedDateRange
     )}/_search`;
-
-    const endpoint = `${endpoints.pbench_server}/elasticsearch`;
+    const endpoint = MOCK_UI
+      ? `${endpoints.pbench_server}/datasets/list`
+      : `${endpoints.pbench_server}/elasticsearch`;
```

Comment:
Ah; interesting. Better to mock based on distinct endpoints rather than the klunky old JSON patterns from the payload; and eventually we can just simplify this to drop the elasticsearch alternative. I like.